webrtc.html

```diff
@@ -783,6 +783,14 @@
                     then reject <var>p</var> with a
                     <code>InvalidStateError</code> and abort these steps.</p>
                   </li>
+
+                  <li> <p>If the content of <var>description</var> is not valid
+                  SDP syntax, then reject <var>p</var> with an
+                  <code>RTCError</code> with errorDetail set to "sdpSyntaxError"
+                  and the sdpLineNumber attribute set to the line number in the
+                  SDP where the syntax error was detected then abort these
+                  steps.</p> </li>
```

Comment:
I still don't see a case where machine action can be taken on the SDP line number. What would change if we instead said "it is RECOMMENDED that the SDP line number be part of the message part of the Error"?